### ISSUE TITLE
How to save worlds in matter-js?

### ISSUE THREAD

**Issue by SharifVelasquezAlzate** [liabru/matter-js#932]
How can I keep the state of my world, even when I refresh the page? I know there's mattertools.serializer, but I don't know how to use it, because documentation doesn't help very much :/. Sorry if my question is a bit obvious or dumb, but I'm a novice using this library, which at the moment has helped my a lot and I think is excellent.

**Comment by goossaert** [comment]
This might help you: https://github.com/liabru/matter-js/issues/569

**Comment by liabru** [comment]
This isn't a core feature of the project but an approach I've used is shown in [MatterTools.Serializer](https://github.com/liabru/matter-tools/blob/master/src/tools/Serializer.js). I realise you've seen it and that code is pretty old but it should still work, essentially it just puts the world object through [resurrect-js](https://github.com/skeeto/resurrect-js) with a few helpers. 

There may be newer libraries around that also handle this but that was one of the best I've found for serialising JS objects in general. It would be nice to move this into a plugin at some point but unfortunately it's not a priority for me at the moment.
